.github/workflows/integration.yml

```diff
@@ -21,6 +21,16 @@ jobs:
       - name: Check out repository
         uses: actions/checkout@v6
 
+      - name: Cache chezmoi downloads
+        uses: actions/cache@v4
+        with:
+          path: |
+            ~/.cache/chezmoi
+            ~/Library/Caches/chezmoi
```

Comment:
Is there an easier way to expect this from the chezmoi CLI itself? That way we don't need as much special case logic per OS